base_layer/wallet/src/output_manager_service/service.rs

```diff
@@ -927,27 +1020,32 @@ where
             "Building coinbase transaction for block_height {} with TxId: {}", block_height, tx_id
         );
 
-        let (spending_key, script_key) = self
+        let spending_key = self
```

Comment:
This is supposed to be a key from the Coinbase branch using the height as the index.